src/libmongoc/src/mongoc/mongoc-server-description-private.h

```diff
@@ -200,4 +200,6 @@ bool
 mongoc_server_description_has_service_id (
    const mongoc_server_description_t *description);
 
+extern bool global_mock_service_id;
```

Comment:
nit: This is a good and pretty unique identifier, but /should/ it be "mongoc_" or some such?
question: Any concerns about concurrent writes to this?